org.jacoco.core.test.validation/pom.xml

```diff
@@ -90,6 +96,9 @@
           <value>7</value>
         </property>
       </activation>
+      <properties>
+        <kotlin.compiler.jvmTarget>1.6</kotlin.compiler.jvmTarget>
```

Comment:
Is this on purpose? No Java 7 target?

Reply:
@marchof This is kinda historical - `kotlin.compiler.jvmTarget` had default value `1.6` and in Kotlin `1.5` default was changed to `1.8`, thus this change doesn't change what we already had before.

Reply:
In this case we need to update `build.html`?

Reply:
See https://github.com/jacoco/jacoco/pull/1181#issuecomment-831841243